File: 17solve/17-156 var1.pas
// Автор В.Н. Шубинкин

##
var a := ReadLines('17-2.txt').Select(t -> t.ToInteger).ToArray;
var maxEl := a.Max;
print(a.CountOf(maxEl), a.IndexMax + 1)

{ Рекомендую также ознакомиться с эффективным однопроходным алгоритмом,
  без использования массива var2}

File: 17solve/17-156 var1.py
# Автор В.Н. Шубинкин

with open('17-2.txt') as f:
    a = [int(x) for x in f.readlines()]

max_el = max(a)
print(a.count(max_el), a.index(max_el) + 1)

'''Рекомендую также ознакомиться с эффективным однопроходным алгоритмом
без использования методов массива и самого массива var2'''
